Code/Editor/PluginManager.cpp

```diff
@@ -332,14 +331,11 @@ IUIEvent* CPluginManager::GetEventByIDAndPluginID(uint8 aPluginID, uint8 aEventI
 
 bool CPluginManager::CanAllPluginsExitNow()
 {
-    for (auto it = m_plugins.begin(); it != m_plugins.end(); ++it)
+    for (const SPluginEntry& m_plugin : m_plugins)
```

Comment:
"m_plugin" should just be "plugin".  The "m_" prefix is reserved for member variables and shouldn't be used for local variables.

Reply:
@mbalfour-amzn Thank you, will update this PR with all suggested fixes.